client/payment-details/dispute-details/dispute-notice.tsx

```diff
@@ -18,52 +19,61 @@ import { isInquiry } from 'wcpay/disputes/utils';
 		__(
 			'The cardholder claims this is an unrecognized charge.',
 			'woocommerce-payments'
 		);
 
-	const noticeText = isInquiry( dispute )
-		? /* translators: <a> link to dispute documentation. %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
-		  __(
-				// eslint-disable-next-line max-len
-				'<strong>%s</strong> You can challenge their claim if you believe it’s invalid. Not responding will result in an automatic loss. <a>Learn more</a>',
-				'woocommerce-payments'
-		  )
-		: /* translators: <a> link to dispute documentation. %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
-		  __(
-				// eslint-disable-next-line max-len
-				'<strong>%s</strong> Challenge the dispute if you believe the claim is invalid, or accept to forfeit the funds and pay the dispute fee. Non-response will result in an automatic loss. <a>Learn more about responding to disputes</a>',
-				'woocommerce-payments'
-		  );
+	/* translators: <a> link to dispute documentation. %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
+	let noticeText = __(
+		'<strong>%s</strong> Challenge the dispute if you believe the claim is invalid, ' +
+			'or accept to forfeit the funds and pay the dispute fee. ' +
+			'Non-response will result in an automatic loss. <a>Learn more about responding to disputes</a>',
+		'woocommerce-payments'
+	);
+	let learnMoreDocsUrl =
+		'https://woocommerce.com/document/woopayments/fraud-and-disputes/managing-disputes/#responding';
+
+	if ( isInquiry( dispute ) ) {
+		/* translators: <a> link to dispute inquiry documentation. %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
+		noticeText = __(
+			'<strong>%s</strong> You can challenge their claim if you believe it’s invalid. ' +
+				'Not responding will result in an automatic loss. <a>Learn more about payment inquiries</a>',
+			'woocommerce-payments'
+		);
+		learnMoreDocsUrl =
+			'https://woocommerce.com/document/woopayments/fraud-and-disputes/managing-disputes/#inquiries';
+	}
 
 	return (
 		<InlineNotice
 			icon
-			status={ urgent ? 'error' : 'warning' }
+			status={ isUrgent ? 'error' : 'warning' }
 			className="dispute-notice"
 			isDismissible={ false }
 		>
-			{ createInterpolateElement( sprintf( noticeText, clientClaim ), {
-				a: (
-					// eslint-disable-next-line jsx-a11y/anchor-has-content
-					<a
-						target="_blank"
-						rel="noopener noreferrer"
-						href="https://woocommerce.com/document/woopayments/fraud-and-disputes/managing-disputes/#section-3"
-					/>
-				),
-				strong: <strong />,
-			} ) }
+			{ createInterpolateElement(
+				sprintf( noticeText, shopperDisputeReason ),
+				{
+					a: (
+						<ExternalLink
+							target="_blank"
+							rel="noopener noreferrer"
+							href={ learnMoreDocsUrl }
+						/>
+					),
+					strong: <strong />,
```

Comment:
I think these `target` and `rel` attributes are redundant, as they're provided by the [ExternalLink component](https://github.com/WordPress/gutenberg/blob/c8fa0db695a1c783dc0bd3d9e91c9547082947c7/packages/components/src/external-link/index.tsx#L60-L68).
```suggestion
					a: <ExternalLink href={ learnMoreDocsUrl } />,
					strong: <strong />,
```

Reply:
Good point! Thanks for catching that.